abi/src/main/java/org/web3j/abi/Utils.java

```diff
@@ -152,4 +161,43 @@ static String getSimpleTypeName(Class<?> type) {
         }
         return result;
     }
+
+    /**
+     * Returns number of canonical fields in a static struct. Example: struct Baz { Struct Bar { int
+     * a, int b }, int c } will return three fields count.
+     *
+     * @param classType
```

Comment:
View comment above.

Also, please remove all empty Javadoc or even better, fill it if possible. 

Eg `@param classType`, `@return` are meaningless.